evasquez26/angular4-startup
# S2-ONE-109 Edit profile

## Story / Task

[ONE-109](https://bixlabs.atlassian.net/browse/ONE-109)


## What this PR does?

Adds functionality to edit my profile information

### Before 
Could not edit ptofile info

### After
User is able to update their information

### How to test / reproduce

- Login to the app
- Using the mocked users data from previuos PRs connect to json-server (this is temporary solution, until we can use the API)
- Go to my profile page
- In the sidebar click, "Edit"
- Inputs for updating the info should appear
- Change data
- Click save

### TODOs

### Related PRs
N/A



## Migrations

NO
